config/base/kustomization.yaml

```diff
@@ -22,16 +24,27 @@ resources:
   - rbac/leader-election.yaml
   - cert-manager/certificate.yaml
 
-vars:
+replacements:
```

Comment:
with the new approach of repeating this replacement in overlay kustomization, there is a slight risk of someone modifying this replacement code, but forgetting to mirror the modification in overlay, in which case the replacement could be buggy, and the tests would still pass.

I don't think there's an elegant way to solve this, but could you add a code comment that the replacement code needs to be mirrored in `config/acceptance/kustomization.yaml`?